src/common/ReusableNavLink.js

```diff
@@ -0,0 +1,11 @@
+import Link from "next/link";
```

Comment:
@Sheyitofunmi What is this change about? Did you relocate this file from the directory it was previously in? If yes, why please?